cmd/thanos/compact.go

```diff
@@ -62,13 +64,14 @@ func (cs compactionSet) levels(maxLevel int) ([]int64, error) {
 	return levels, nil
 }
 
-// maxLevel returns max available compaction level
+// maxLevel returns max available compaction level.
 func (cs compactionSet) maxLevel() int {
 	return len(cs) - 1
 }
 
-func registerCompact(m map[string]setupFunc, app *kingpin.Application, name string) {
-	cmd := app.Command(name, "continuously compacts blocks in an object store bucket")
+func registerCompact(m map[string]setupFunc, app *kingpin.Application) {
+	component := component.Compact
```

Comment:
`const component = component.Compact`?

Reply:
hm it looks like struct or any other more complex data structure cannot be a constant IIUC?
I renamed the variable at least to `comp` so it does not collide with the package name.